### PR TITLE
Updates for release 1.0.3

### PR DESCRIPTION
**Pull request by fredtibbitts**
PR with updates for the 1.0.3 patch release and fixes to commands to include the verrazzano-install namespace.

**NOTE**: The patch release date in the versions.md file must be updated with the release date before this PR is merged.

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ weight: 13
 draft: false
 ---
 
+### v1.0.3
+Fixes:
+- Fix to use load balancer service external IP for application ingress when using an external load balancer and wildcard DNS.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: external IP -> external IP address

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ weight: 13
...
+- Rebuilt several component images to address known issues.
+- Updated to the following versions:
+    - Grafana 6.7.4.
+    - MySQL 8.0.27
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:14]: MySQL 8.0.27 -> MySQL 8.0.27. (period at the end)

**Review by rosemarymarano — COMMENTED** [review]
Minor edits.

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (fredtibbitts) [reply content/en/docs/releasenotes/_index.md]: Fixed.

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (fredtibbitts) [reply content/en/docs/releasenotes/_index.md]: We just reverted the MySQL upgrade, so I removed this completely.

**Review by rosemarymarano — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]

**Review by markxnelson — APPROVED** [review]

**Pull request merged**